route/middleware.go

```diff
@@ -13,7 +13,7 @@ import (
 
 // for generating request IDs
 func init() {
-	rand.Seed(time.Now().UnixNano())
+	rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
What's the change for?

Reply:
Go's stdlib changed the way random numbers worked; the original version of this is no longer functional.